apps/parse/lib/parse/calendar_attributes.ex

```diff
@@ -28,13 +34,26 @@ defmodule Parse.CalendarAttributes do
       description: copy_string(row["service_description"]),
       schedule_name: copy_string(row["service_schedule_name"]),
       schedule_type: copy_string(row["service_schedule_type"]),
-      schedule_typicality: copy_int(row["service_schedule_typicality"])
+      schedule_typicality: copy_int(row["service_schedule_typicality"]),
+      rating_start_date: copy_date(row["rating_start_date"]),
+      rating_end_date: copy_date(row["rating_end_date"]),
+      rating_description: copy_string(row["rating_description"])
     }
   end
 
+  defp copy_string(nil), do: nil
   defp copy_string(""), do: nil
   defp copy_string(s), do: :binary.copy(s)
 
   defp copy_int(""), do: nil
   defp copy_int(s), do: String.to_integer(s)
+
+  defp copy_date(nil), do: nil
+  defp copy_date(""), do: nil
+
+  defp copy_date(s) do
+    s
+    |> Timex.parse!("{YYYY}{0M}{0D}")
+    |> NaiveDateTime.to_date()
+  end
```

Comment:
Can we do this without adding a new dependency on Timex?
```elixir
defp copy_date(<< year_bin::binary-4, month_bin::binary-2, day_bin::binary-2 >>) do
  Date.new(String.to_integer(year_bin), String.to_integer(month_bin), String.to_integer(day_bin))
end
```

Reply:
We're currently using Timex to parse similarly formatted dates in at least `calendar.txt`, `calendar_dates.txt` and `feed_info.txt` -- should I change all of them?

Reply:
No, nevermind; I thought we'd done more work to remove Timex but must have misremembered.